src/Exchange/GwfGwfExchange.f90

```diff
@@ -2,7 +2,7 @@ module GwfGwfExchangeModule
 
   use KindModule, only: DP, I4B
   use SimVariablesModule, only: errmsg
-  use ArrayHandlersModule,     only: ExpandArray
+  !use ArrayHandlersModule,     only: ExpandArray
```

Comment:
Obviously, would be nice to remove all this old commented code.

Reply:
commented code removed in latest pull request commit